packages/react-server-cli/src/defaultOptions.js

```diff
@@ -0,0 +1,16 @@
+// these are the default options for startServer.
```

Comment:
Nice to have this pulled out into a dedicated module!  Also nice to have consistent defaults between the CLI and a direct call `startServer`.